src/screens/SendToken/SendTokenPinConfirmScreen.js

```diff
@@ -88,11 +92,14 @@ class SendTokenPinConfirmScreen extends React.Component<Props, State> {
       });
       return;
     }
+    const senderAccountAddress = getActiveAccountAddress(accounts);
+    const { to: recipient } = transactionPayload;
 
     this.setState({
       isChecking: true,
     }, () => {
       logEvent('transaction_sent', { source: this.source });
+      isLogV2AppEvents() && logEvent('v2_transaction_sent', { to: recipient, from: senderAccountAddress });
```

Comment:
Sorry, i don't think i made this clear, but here the `from` and `to` need to be the `from` and `to` asset symbol (like PLR)

Reply:
@IAmKio Done